packages/babel-plugin-transform-modules-commonjs/test/fixtures/interop/export-all/input.js

```diff
@@ -0,0 +1,5 @@
+// The fact that it exports both a default, _and_ all of the names via re-export
```

Comment:
would be good to actually describe those edge cases (or link to some issue describing them)